src/pages/iou/SplitBillDetailsPage.js

```diff
@@ -143,7 +143,7 @@ function SplitBillDetailsPage(props) {
                             hasSmartScanFailed={hasSmartScanFailed}
                             reportID={reportID}
                             reportActionID={reportAction.reportActionID}
-                            transactionID={props.transaction.transactionID}
+                            transaction={isEditingSplitBill ? props.draftTransaction : props.transaction}
```

Comment:
This line caused https://github.com/Expensify/App/issues/32809, more details https://github.com/Expensify/App/issues/32809#issuecomment-1850231768